src/types/venues.ts

```diff
@@ -145,6 +145,7 @@ export interface BaseVenue {
   hideVideo?: boolean;
   showLiveSchedule?: boolean;
   showGrid?: boolean;
+  showInfo?: boolean;
```

Comment:
Should we also be implementing the toggle in the admin tooling to be able to enable/disable this setting?